components/turret.py

```diff
@@ -1,42 +1,91 @@
-class Turret:
-    # GEAR_RATIO: float
-    # ANGLE_ERROR_TOLERANCE: float
+from math import pi, radians
+from rev import CANSparkMax
+from wpilib import DigitalInput
+from wpimath.controller import ProfiledPIDControllerRadians
+from wpimath.trajectory import TrapezoidProfileRadians
+from ids import SparkMaxIds, DioChannels
+
+from  magicbot import tunable, feedback
+
 
-    # motor: Neo550 #This will need to change to the proper motor controller type
-    # positive_limit_switch: Digital I/O
-    # negative_limit_switch: Digital I/O # These could also be attached directly to the motor controller breakout board for interrupts
+GEAR_RATIO: float = 18/140
```

Comment:
```suggestion
GEAR_RATIO: float = (7 / 1) * (4 / 1) * (18/140)
```
This is the current gear ratio from the arm gearbox we pilfered